interpretation/ww_head.py

```diff
@@ -985,23 +993,28 @@ def process(message, isGameMaster=False, isAdmin=False, isPeasant=False):
             '''powder'''
             # Powder a player
             if is_command(message, ['creeper', 'powder']) and user_role == "Pyromancer":
-                # TODO
-                return todo()
-            if is_command(message, ['creeper', 'powder'], True) and user_role == "Pyromancer":
-                # TODO
-                target = check.users(message, amount=1, must_be_participant=True)
+                target = check.users(message,1,True,True)
                 if not target:
-                    return [Mailbox().embed(destination=message.channel, embed=Embed(
-                    title="Invalid user", description="Please mention a user (either by mention, or by emoji)"
-                        .format(user=message.author.id, target=target[0].id)), temporary=True)]
-
-                func.powder(message.author.id, target[0].id)
-                return [Mailbox().embed(destination=message.channel, embed=Embed(
-                    title="Powdered a participant", description="<@{user}> powdered <@{target}>"
-                        .format(user=message.author.id, target=target[0].id)))]
+                    return [Mailbox().respond("No target provided! Please give us a target to powder.",True)]
+                return [func.powder(user_id,target[0])]
+            if is_command(message, ['creeper', 'powder'], True) and user_role == "Pyromancer":
+                msg = "**Usage:** Powder a player.\n\n`" + prefix + "powder <target>`\n\n"
+                msg += "**Example:** `" + prefix + "powder @Randium#6521`\nThis command can only be used by pyromancers."
+                return [Mailbox().respond(msg,True)]
             if user_role == "Pyromancer":
                 help_msg += "`" + prefix + "powder` - Powder a player. (Pyromancer only)\n"
 
+            '''ignite'''
+            # Set all powdered players on fire.
+            if is_command(message,['ignite','flame','blaze']) and user_role == "Pyromancer":
+                return [func.ignite(user_id)]
```

Comment:
Why do we ignite with a given user_id? are powders bound to a certain user? (like if we have two pyros, have to seperate kill queues). when i played two pyros, it wasnt like that

Reply:
They are bound to the caster. The function `ìgnite()` needs to know which pyromancer has cast the ignite effect. This prevents one pyromancer from powdering a player and then stealing the other pyro's power to ignite all players.

The `user_id` value is the pyromancer's id, not the victim's.